MatomoTracker/MatomoTracker.swift

```diff
@@ -468,3 +468,24 @@ extension MatomoTracker {
     /// The version of the Matomo SDKs
     @objc public static let sdkVersion = "7.3"
 }
+
+extension MatomoTracker {
+    /// Resets all session, visitor and campaign information.
+    ///
+    /// Dispatches events before restting itself.
+    /// After calling this method this instance behaves like the app has been freshly installed.
+    public func reset() {
```

Comment:
I think, first think in this function, we should ensure we are running on the main thread.